DigitalExcellence/dex-frontend
# Typo fix: Rename hightlight.ts to highlight.ts

## Description

This has been bugging me for a while: The file `hightlight.ts` should be renamed to `highlight.ts`.

## Type of change


## Checklist


## Steps to Test or Reproduce

Outline the steps to test or reproduce the PR here.
These steps will be used during release testing.

1. Build source code
2. Verify that it still works

## Link to issue

None
